joplin/settings.py

```diff
@@ -271,6 +271,8 @@
     DEFAULT_FILE_STORAGE = 'custom_storages.MediaStorage'
     DBBACKUP_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
 
+JANIS_SLUG_URL=""
```

Comment:
What's this for? I know we use this url to trigger rebuild of janis when a page is unpublished or published on staging and prod, if all the settings variables are defined in signals.py (below).
```
if not all([settings.HEROKU_KEY, settings.HEROKU_JANIS_APP_NAME]):
```
should we just add `setting.JANIS_SLUG_URL` to the array above instead of making an empty definition?